src/utils/field-utils.js

```diff
@@ -0,0 +1,21 @@
+/* eslint import/prefer-default-export: "off" */
+
+/**
+ * Retrieves the model to which the supplied field refers.
+ *
+ * @param {Object} field - The field definition to determine the ref for
+ * @returns {Model} - The model to which the field refers
+ */
+export function getRef(field) {
+    if (!field.ref) {
+        return null;
+    }
+
+    if (field.ref._modelMeta) {
+        return field.ref;
+    }
+    if (Object.prototype.toString.call(field.ref) === '[object Function]') {
```

Comment:
What's this doing exactly? Just checking if its a function? Is it different from field.ref.toString() ?